NetKAN/AtmosphereAutopilot.netkan

```diff
@@ -5,7 +5,7 @@
     "name"         : "AtmosphereAutopilot (Fly-By-Wire)",
     "abstract"     : "Atmosphere autopilot is a modular atmospheric flight control system library.\nIt features a Fly-By-Wire System that replaces the stock SAS, and cruise control with heading/altitude/velocity setpoints. All without tuning PID parameters!\n\nIf you're using Stock Bug Fix Modules, turn off ModuleControlSurfacePlus and ModuleGimbalPlus modules in its manager.",
     "license"     :  "GPL-3.0",
-    "ksp_version"  : "1.0.5",
+    "ksp_version"  : "1.1.0",
```

Comment:
Replace this line with `"$vref": "#/ckan/ksp-avc",` and we'll be able to scrape compatibility information from the .version file. No more manual updates to the .netkan with each new KSP version.